cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -115,11 +115,21 @@ jobs:
       - uses: swatinem/rust-cache@v2
       - name: Install cargo-dist
         run: ${{ matrix.install_dist }}
+      - name: Install dependencies
+        run: |
+          ${{ matrix.packages_install }}
       - name: Build artifacts
+        if: ${{ hashFiles('Brewfile') == '' }}
         run: |
           # Actually do builds and make zips and whatnot
           cargo dist build ${{ needs.plan.outputs.tag-flag }} --output-format=json ${{ matrix.dist_args }} > dist-manifest.json
           echo "cargo dist ran successfully"
+      - name: Build artifacts (using Brewfile)
```

Comment:
The duplication here is because Windows Powershell doesn't support the syntax that would have let me do the `$CARGO dist blah` exactly the same way as it works on bash. It worked out to be easier to just duplicate this with slightly different logic for now.